### PR TITLE
Gulp / Webpack bundling

### PR DESCRIPTION
**Pull request by bmbergh**
Gulp and webpack are now bundling correctly

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+	sourcemaps = require('gulp-sourcemaps'),
+	gutil = require("gulp-util"),
+	webpack = require("webpack"),
+	WebpackDevServer = require("webpack-dev-server"),
+	webpackConfig = require("./webpack.config.js"),
+	stream = require('webpack-stream'),
+	nodemon = require('gulp-nodemon'),
+	jshint = require('gulp-jshint'),
+	stylish = require('jshint-stylish'),
+	chalk = require('chalk');
```
Comment (al1271) [inline gulpfile.js]: editor spacing issue

File: gulpfile.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+	myConfig.debug = true;
+
+	// Start a webpack-dev-server
+	new WebpackDevServer(webpack(myConfig), {
+		publicPath: myConfig.output.publicPath,
+		stats: {
+			colors: true
+		}
+	 });
+});*/
```
Comment (al1271) [inline gulpfile.js]: commented out code is fine in dev branch but don't leave it for master

File: gulpfile.js [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (al1271) [inline gulpfile.js]: white space

File: client/main.jsx [hunk 4]
```diff
@@ -0,0 +1,9 @@
+/*
+*
+* 
+Main React component - this is where we will create the view controller for displaying our dashboard 
+*
+*
+*/
+//You will need to make sure that you have react-dom.
+//With react 14.+ they added this react-dom module to handle dom rendering
```
Comment (al1271) [inline client/main.jsx]: empty file?

**Comment by bmbergh** [comment]
@al1271 @karhodes  I fixed the node modules and the dependancies. The package.json file does not have carats so all version numbers are the same.

**Pull request merged**